### PR TITLE
Create MariaDB system user for /var/lib/mariadb, bsc#1034885

### PR DESCRIPTION
**Pull request by mjura**
Create and reserve UID/GID for user MariaDB to own /var/lib/mariadb
for security reasons

### CODE REVIEW

**Comment by flavio** [comment]
To be put on hold until we agree on how to address bsc#1034885

File: activate.sh [hunk 1]
```diff
@@ -46,3 +46,7 @@ if ! [ -f /root/.ssh/id_rsa ]; then
 fi
 [ -d /var/lib/misc/ssh-public-key  ] || mkdir -p /var/lib/misc/ssh-public-key
 cp /root/.ssh/id_rsa.pub /var/lib/misc/ssh-public-key
+
+# Create MariaDB system user, bsc#1034885
+groupadd -g 331 mysql
+useradd -r -u 331 -g mysql -c "MySQL database admin" -d /var/lib/mysql -s /bin/false mysql
```
Comment (flavio) [inline activate.sh]: Where does these numbers come from? Are these the ones used by default by the mariadb package on openSUSE/SLE?

**Comment by PI-Victor** [comment]
any updates on the workflow, bugzilla discussion didn't progress passed this PR.

File: activate.sh (hunk 1 above)
Comment (jordimassaguerpla) [inline activate.sh]: You better use uid 60 for mysql user, which is the one used by mariadb user:

See: https://build.suse.de/package/view_file/SUSE:SLE-12-SP2:Update/mariadb/mariadb.spec?expand=1

You can see in the pre section this:

getent group mysql >/dev/null || groupadd -r mysql
getent passwd mysql >/dev/null || useradd -r -o -g mysql -u 60 -c "MySQL database admin" \
                  -s /bin/false -d %{_localstatedir}/lib/mysql mysql
usermod -g mysql -s /bin/false mysql

Comment (jordimassaguerpla) [inline activate.sh]: Group does not have a gid in the package so you cannot know in advance which will be choosen in the mariadb image, but you can be certain about the uid 60 .

Comment (mjura) [inline activate.sh]: @jordimassaguerpla : Thank you for your findings about mariadb. For GID we can modify kiwi mariadb container template and add mysql group in advance with fixed GID.

**Comment by mjura** [comment]
@PI-Victor : I've fixed comments and rebased this change

**Comment by PI-Victor** [comment]
anyone know - can we proceed with merging this? or are we still waiting for the other bug to be solved?

**Comment by flavio** [comment]
We have put this PR on hold, we need to agree about how to fix that on bugzilla.

**Comment by ereslibre** [comment]
What's the status of this, should we close this PR?

**Pull request closed without merging**